### PR TITLE
DBotPredictURLPhishing - timeout argument

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CRTX-87358)

## Description
Added the rasterize_timeout argument to the script

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py [hunk 1]
```diff
@@ -474,13 +474,13 @@ def extract_created_date(entry_list: list):
     return None
 
 
-def get_prediction_single_url(model, url, force_model, who_is_enabled, debug):
+def get_prediction_single_url(model, url, force_model, who_is_enabled, debug, rasterize_timeout):
```
Comment (daryakoval) [inline Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py:477]:
```suggestion
def get_prediction_single_url(model, url, force_model, who_is_enabled, debug, rasterize_timeout=TIMEOUT_RASTERIZE):
```

**Review by daryakoval — APPROVED** [review]
Great :)

File: Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py (hunk 1 above)
Comment (MLainer1) [reply Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py:477]: The value for the argument *rasterize_timeout* is decided by:
`arg_to_number(demisto.args().get('rasterize_timeout', TIMEOUT_RASTERIZE))`
in the main, so I do not think it is necessary here

**Comment by MLainer1** [comment]
This PR needs a force-merge, as updating the Docker image (required validation) breaks ML-related integrations (known issue).

```
Error with updated Docker:

Failed to execute URL Phishing script. Error: 'ColumnTransformer' object has no attribute '_name_to_fitted_passthrough'

```
[StackOverflow](https://stackoverflow.com/questions/75462901/attributeerror-columntransformer-object-has-no-attribute-name-to-fitted-pas)

**Pull request merged**